src/actions/channels.js

```diff
@@ -117,6 +117,10 @@ export function createDirectChannel(userId, otherUserId) {
             last_update_at: created.create_at
         };
 
+        const preference = [{category: Preferences.CATEGORY_DIRECT_CHANNEL_SHOW, name: otherUserId, value: 'true'}];
+
+        savePreferences(userId, preference);
```

Comment:
note to self: remove the save preference in the RN action when creating a direct channel as is going to be handled by this